server/test/services/role/RoleServiceTest.java

```diff
@@ -151,6 +152,55 @@ public void makeProgramAdmins_manyEmailsHaveNoAccountReturnsError()
                         email1, email2))));
   }
 
+  @Test
+  public void makeProgramAdmins_blockGlobalAdmin() throws ProgramNotFoundException {
+    String globalAdminEmail = "global@admin";
+    Account globalAdmin = new Account();
+    globalAdmin.setEmailAddress(globalAdminEmail);
+    globalAdmin.setGlobalAdmin(true);
+    globalAdmin.save();
+
+    String programName = "test program";
+    Program program = ProgramBuilder.newDraftProgram(programName).build();
+
+    RoleService serviceWithGlobalAdminDisabled =
+        new RoleService(instanceOf(ProgramService.class), instanceOf(UserRepository.class), false);
```

Comment:
Style nit: It's pretty tough to find the place these two tests diverge. Consider either pulling out two constants `ALLOWS_PROGRAM_ADMINS_AS_CIVIFORM_ADMINS = true` or add comment-strings indicating the parameter:

`instanceOf(UserRepository.class), /* allowGlobalAdmins= */ false);`

Same for the below test.